src/components/todos.js

```diff
@@ -0,0 +1,30 @@
+/* eslint react/prop-types: 0 */
+import React from 'react';
+import PropTypes from 'prop-types';
+import TodoItem from './elements';
+
+class TodosList extends React.PureComponent {
+  render() {
+    const { todos, handleChangeProps, deleteTodoProps } = this.props;
+    return (
+      <ul>
+        {todos.map((todo) => (
+          <TodoItem
+            key={todo.id}
+            todo={todo}
+            handleChangeProps={handleChangeProps}
+            deleteTodoProps={deleteTodoProps}
+          />
+        ))}
+      </ul>
+    );
+  }
+}
+
+TodosList.propTypes = {
+  todos: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
There is a linting error regarding the prototypes due to the non-specific selector function. Kindly fix this issue.